projects/driver_controls/inc/ads1015_def.h

```diff
@@ -0,0 +1,108 @@
+
+// ********* The following bytes are fields for the config register *********************
+// ********* From section 8.6.3 of the datasheet ****************************************
+
+// Starts single conversion when in powerdown state
+#define ADS1015_START_SINGLE_CONV ((uint8_t)0x1 << 7)
+
+// Bytes for setting channels
+#define ADS1015_AIN(channel) ((channel + 0x4) << 4)
+#define ADS1015_AIN_0 ((uint8_t)0x4 << 4)
+#define ADS1015_AIN_1 ((uint8_t)0x5 << 4)
+#define ADS1015_AIN_2 ((uint8_t)0x6 << 4)
+#define ADS1015_AIN_3 ((uint8_t)0x7 << 4)
+
+// These bytes set the FSR of the programmable gain amplifier
+#define ADS1015_PGA_FSR_6144 ((uint8_t)0x0 << 1)  // ±6.144 V
+#define ADS1015_PGA_FSR_4096 ((uint8_t)0x1 << 1)  // ±4.096 V (default)
+#define ADS1015_PGA_FSR_2048 ((uint8_t)0x2 << 1)  // ±2.048 V
+#define ADS1015_PGA_FSR_1024 ((uint8_t)0x3 << 1)  // ±1.024 V
+#define ADS1015_PGA_FSR_512 ((uint8_t)0x4 << 1)   // ±0.512 V
+#define ADS1015_PGA_FSR_256 ((uint8_t)0x5 << 1)   // ±0.256 V
+
+// Conversion mode (single or continuous)
+#define ADS1015_CONVERSION_MODE_CONT ((uint8_t)0x0)
+#define ADS1015_CONVERSION_MODE_SINGLE ((uint8_t)0x1)  // default
+
+// Data sampling rate of the adc
+#define ADS1015_DATA_RATE_128 ((uint8_t)0x0 << 5)   //  128 SPS
+#define ADS1015_DATA_RATE_250 ((uint8_t)0x1 << 5)   //  250 SPS
+#define ADS1015_DATA_RATE_490 ((uint8_t)0x2 << 5)   //  490 SPS
+#define ADS1015_DATA_RATE_920 ((uint8_t)0x3 << 5)   //  920 SPS
+#define ADS1015_DATA_RATE_1600 ((uint8_t)0x4 << 5)  // 1600 SPS
+#define ADS1015_DATA_RATE_2400 ((uint8_t)0x5 << 5)  // 2400 SPS
+#define ADS1015_DATA_RATE_3300 ((uint8_t)0x6 << 5)  // 3300 SPS
+
+// Comparator operating mode
+#define ADS1015_COMP_MODE_TRAD ((uint8_t)0x0 << 4)  // default
+#define ADS1015_COMP_MODE_WINDOW ((uint8_t)0x1 << 4)
+
+// Polarity of the ALERT/RDY pin (active low or active high)
+#define ADS1015_COMP_POL_LOW ((uint8_t)0x0 << 3)
+#define ADS1015_COMP_POL_HIGH ((uint8_t)0x1 << 3)
+
+// Latching comparator
+#define ADS1015_COMP_LAT_NON_LATCHING ((uint8_t)0x0 << 2)
+#define ADS1015_COMP_LAT_LATCHING ((uint8_t)0x1 << 2)
+
+// Comparator queue and disable
+#define ADS1015_COMP_QUE_1_CONV ((uint8_t)0x0)  // Assert after one conversion
+#define ADS1015_COMP_QUE_2_CONV ((uint8_t)0x1)  // Assert after two conversion
+#define ADS1015_COMP_QUE_4_CONV ((uint8_t)0x2)  // Assert after four conversion
+#define ADS1015_COMP_QUE_DISABLE_COMP \
+  ((uint8_t)0x3)  // Disable comparator and set ALERT/RDY pin to high-impedance
+
+// **************************************************************************************
+
+// I2C general call address from section 8.5.1.2 of the datasheet
+#define ADS1015_I2C_GENERAL_CALL ((uint8_t)0x0)
+
+// Reset command byte from section 8.5.1.2 of the datasheet
+#define ADS1015_RESET_BYTE ((uint8_t)0x6)
+
+// Setup for the config register's upper byte.
+#define CONFIG_REGISTER_MSB(channel)                                         \
```

Comment:
Can you make sure all your #defines match the naming scheme? (start with `ADS1015_`)